Assembler.Base/FrameHandlers/MiddleFrameHandler.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Assembler.Core;
+using Assembler.Core.Entities;
+
+namespace Assembler.Base.FrameHandlers
+{
+    public class MiddleFrameHandler<TFrame, TMessageInAssembly> : BaseFrameHandler<TFrame, TMessageInAssembly>
```

Comment:
See comment of InitialFrameHandler and apply here

Reply:
f